common/test/acceptance/pages/lms/teams.py

```diff
@@ -102,3 +103,42 @@ def header_topic_description(self):
     def team_cards(self):
         """Get all the team cards on the page."""
         return self.q(css='.team-card')
+
+
+class TeamPage(CoursePage, PaginatedUIMixin):
+    """
+    The page for a specific Team within the Teams tab
+    """
+    def __init__(self, browser, course_id, team):
+        """
+        Set up `self.url_path` on instantiation, since it dynamically
```

Comment:
I don't think this documentation is correct. It should talk about "team" instead of "topic".